scripts/simplify_network.py

```diff
@@ -620,6 +706,11 @@ def consense(x):
 
     update_p_nom_max(n)
 
+    e_threshold_drop_isolated = cluster_config.get("e_threshold_drop_isolated ", False)
+    if e_threshold_drop_isolated:
+        drop_isolated_nodes(n, drop_e_threshold)
+        n.determine_network_topology()
```

Comment:
The update of the determine topology may be moved within ghr drop isolated function

Reply:
Done

Reply:
What I don't understand is why filtering isolated buses `n.buses[(~n.buses.duplicated(subset=["sub_network"], keep=False))]` gives an empty output unless `n.determine_network_topology()` is called before that

Reply:
That function fills the sub_network information, that's why it is needed to have the value well calculated